ztp/CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Getting started
+
+Thanks for asking on how to contribute! Let's get you started!
+
+* Subscribe to our [mailing list](https://cnf_features_deploy_mailing.list)
```

Comment:
I don't think there is a public mailing list or slack channel for ztp, so should delete these lines.

Reply:
Hi Bart, thanks for commenting. I left that blank intentionally as a matter of getting traction as I wasn't sure about if there was any list/slack channel. I'll remove the list. As per slack channel, should I mention any specifically?